stack/building-pipelines-lsst-io-with-documenteer-job.rst

```diff
@@ -0,0 +1,95 @@
+#########################################################################
+Building pipelines.lsst.io with Jenkins (sqre/infrastructure/documenteer)
+#########################################################################
+
+The `sqre/infrastructure/documenteer`_ Jenkins CI job enables you to create a new build of `pipelines.lsst.io`_.
+Since `pipelines.lsst.io`_ is published through LSST the Docs, builds for development branches appear as new editions from the `pipelines.lsst.io/v <https://pipelines.lsst.io/v>`__ version dashboard.
+
+.. important::
+
+   You can't use this Jenkins CI job to test development branches of packages with the `pipelines.lsst.io`_ site build.
+   This Jenkins CI job always uses packages corresponding to a released EUPS tag (such as a daily, weekly, or stable release).
+   Only the branch of the `pipelines_lsst_io`_ repository can be modified.
+
+   If you need to build and test your package’s documentation, you can do so locally with these methods:
+
+   - :doc:`building-pipelines-lsst-io-locally`
+   - :doc:`building-single-package-docs`
+
+.. _jenkins-pipelines-lsst-io-build-running:
+
+Running the documenteer Jenkins CI job
+======================================
+
+Go to the `sqre/infrastructure/documenteer`_ job’s page, then click the **Run** button.
+
+The following sections describe how to set each form field.
+
+.. _jenkins-pipelines-lsst-io-build-eups-distrib-tag-name:
+
+EUPS distrib tag name
+---------------------
+
+This is the EUPS tag of the ``lsst_distrib`` Stack.
+You can set this to a stable release, weekly release, or a daily release.
+
+This means that you can only build documentation for the ``master`` branches of Stack packages that have been released by Jenkins CI.
+
+.. _jenkins-pipelines-lsst-io-build-ltd-edition-slug:
+
+LTD edition slug
+----------------
+
+Set this field to either a Git ref or an EUPS tag.
+Follow these guidelines:
+
+- If you are building with a non-\ ``master`` Git ref of the `pipelines_lsst_io`_ repository (see :ref:`jenkins-pipelines-lsst-io-build-github-repo-ref`), set this field to the name of the `pipelines_lsst_io`_ branch being built.
```

Comment:
Are there changes to the branch name that need to be made?  E.g. `tickets/DM-NNNNN` becomes `DM-NNNNN` or `u/ktl/something` becomes `u-ktl-something`?  Or is this automatically handled?  If the latter, then is there ever any difference between this field and "git repo ref"?  If not, do we need both?